org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/ProcessParameterPanel.java

```diff
@@ -654,6 +654,14 @@ public boolean saveParameters() {
 			//Save only parameters which are set
 			if((result == null) && (result2 == null))
 				continue;
+			if(result instanceof String) {
+				if((result2 == null) && (Util.isEmpty(String.valueOf(result))))
```

Comment:
this can be simplify to Util.isEmpty((String)result). Note that the (String) cast is safe here since it is inside the "instanceof String" block.